tests/code/test_userManagement.py

```diff
@@ -119,8 +119,8 @@ def test_getAllowedPrograms():
     with mainDB.atomic() as transaction:
         # checks the length of all programs an admin has access to and compares that to total programs
         allowedPrograms = len(getAllowedPrograms(User.get_by_id("ramsayb2")))
-        totalPrograms = Program.select().count()
-        assert allowedPrograms == totalPrograms
+        NonOtherPrograms = Program.select().count() - 1 # - Other Celts-Sponsored
```

Comment:
I do not like the variable name `NonOtherPrograms`. the first letter should not be capital and the comment suggests that this is for the other CELTS program so why is programs plural in the variable?